scripts/sct_deepseg.py

```diff
@@ -115,7 +115,18 @@ def main():
 
     # Deal with model
     if args['list_models']:
-        sct.deepseg.models.list_models()
+        models = sct.deepseg.models.list_models()
+
+        color = {True: '\033[92m', False: '\033[91m'}
+        default = {True: '[*]', False: ''}
+        print("{:<25s}DESCRIPTION".format("MODEL"))
+        print("-" * 80)
+        for name_model, value in models.items():
+            print("{}{:<25s}{}\033[0m".format(color[sct.deepseg.models.is_installed(name_model)],
```

Comment:
Oh hang on, maybe I didn't think my suggestion through. If `deepseg.models.list_models()` only returns installed models then `is_installed()` is redundant here. Did you want to also show the available models in the UI too?

Reply:
> If deepseg.models.list_models() only returns installed models then is_installed() is redundant here. Did you want to also show the available models in the UI too? That would make sense.

the latter is the implemented behaviour, i.e., it shows the installed (in green) and not-installed (in red).

Reply:
> the latter is the implemented behaviour, i.e., it shows the installed (in green) and not-installed (in red). And the default models have suffix "[*]"

oopsi! nope, there is a bug-- only the installed are shown-- fixing it now

Reply:
fixed in 79a5fb3de38ccc6c4ae029a8666f2411b7616141